content/archive/internal/paulgonzaloparedes-derecho-de-daos.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Máster en derecho de daños"
-author: "Paul Paredes1 Pontificia Universidad Católica del Perú"
+author: "Paul Paredes1 \(^{1,2}\),\(^1\)Universidad de Girona, \(^2\)Pontificia Universidad Católica del Perú"
```

Comment:
Sounds like your fix didn't work? https://github.com/rstudio/bookdown.org/blob/043c15e8399e4dddd04262edbf4c069f31ec7c7f/R/create-md.R#L204

Reply:
It is because it is cached I believe. So the fix does not apply as book is not processed.

Reply:
I see. I think you could move the processing down to this line: https://github.com/rstudio/bookdown.org/blob/043c15e8399e4dddd04262edbf4c069f31ec7c7f/R/create-md.R#L341

Reply:
I wanted to put it with other special cases we found. I think it works ok now. 
If I have more issue, I'll do the processing later. Thanks!